### DISCUSSION TITLE
[水帖]有没有自制「萝卜」的教程?

### DISCUSSION THREAD

**Discussion by APEX-114514** [share121/inter-knot#1094] [其他]
我拿到了学校的开源空洞数据(具体信息不方便透露)，想试试兼职做个绳匠，第一步当然要从自制「萝卜」开始做起啦。放心我不坑人，只想拿一些低危空洞练练手。虽然网上也能搜到自制「萝卜」的教程，但还是想问问大家在制作「萝卜」这方面有什么心得?
![7c22c2002e9d8de9f0918ede6ebd74c](https://github.com/user-attachments/assets/5b11394f-e9d2-4c0d-a626-61efe30090d5)

**Comment by xiubachi** [comment]
插眼，我也想学

**Comment by tyhpoi** [comment]
坏了，遇见新艾莉都本地人了（⊙ｏ⊙）

**Comment by fairy-B** [comment]
亲，这边建议用虚狩之一开发的HDD和邦布进行感官同步后小小的探索一下，只不过可能有点废邦布
